### DISCUSSION TITLE
How to add rows to Supabase Database after calling a webhook?

### DISCUSSION THREAD

**Discussion by cwilharm** [supabase/supabase#10788] [Questions]
Hello to all!

I am trying to create a webhook with Supabase. After the customer buys an item on my website, I want a webhook to be called to add a row to my Supabase database. How can I create webhooks like that or what do I have to do?

I am not very sure if I have worded myself correctly.... 

Thanks to all in advance!

Constantin

**Comment by GaryAustin1** [comment]
Can you just call the Supabase API, or the direct database port to do the insert to the database?  You could also call a Postgres database function either way if you have some processing to do, or multiple tables to insert/update.

I guess it depends on what your website has for code as to what you can send out.
